Department-for-Transport-Disruptions/create-disruptions-data
# Restricted user to publish disruptions using pending approval status

Testing instructions:

Click create new disruptions in dashboard page and proceed upto Review Disruption page. Restricted users should be displayed "Send to review" button instead of "Publish" button. The same is applicable when disruption is edited in disruption detail page
